app/client/src/git/sagas/deleteBranchSaga.ts

```
@@ -35,12 +37,19 @@ export default function* deleteBranchSaga(
         }),
       );
     }
-  } catch (error) {
-    yield put(
-      gitArtifactActions.deleteBranchError({
-        ...basePayload,
-        error: error as string,
-      }),
-    );
+  } catch (e) {
+    if (response && response.responseMeta.error) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)